prow/plank/controller.go

```diff
@@ -554,3 +557,29 @@ func getPodBuildID(pod *coreapi.Pod) string {
 	logrus.Warningf("BUILD_ID was not found in pod %q: streaming logs from deck will not work", pod.ObjectMeta.Name)
 	return ""
 }
+
+func (c *Controller) patchProwjob(srcPJ prowapi.ProwJob, destPJ prowapi.ProwJob) error {
+	srcPJData, err := json.Marshal(srcPJ)
+	if err != nil {
+		return fmt.Errorf("error marshal srcPJ: %v", err)
```

Comment:
These errors do not expose enough about what is going on here

Reply:
do you want to append the entire prowjob here?

Reply:
No, that's not useful either. Something like "could not create JSON patch:". No admin knows what `srcPJ` is

Reply:
done